app/asr/third_party/warpctc/CMakeLists.txt

```diff
@@ -6,53 +6,87 @@ ENDIF()
 
 project(ctc_release)
 
-IF (NOT APPLE)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp -O2")
-ENDIF()
-
-IF (APPLE)
-  # NOTE: setting -std this again causes nvcc to break, since this is already
-  # set in flashlight/cmake/CUDAUtils.cmake::set_cuda_cxx_compile_flags
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
-    # set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -O2")
-    add_definitions(-DAPPLE)
-ENDIF()
 
 include_directories(include)
 
 FIND_PACKAGE(CUDA 6.5)
-MESSAGE(STATUS "warpctc: cuda found ${CUDA_FOUND}")
+FIND_PACKAGE(Torch)
```

Comment:
Would it be possible to disable looking Torch up front or add `QUIET` to this at the very least? It would probably confuse the user quite a lot if the build looks for Torch.